server/lib/flow-event.js

```diff
@@ -226,8 +226,10 @@ function logFlowEvent (event, data, request) {
   optionallySetFallbackData(eventData, 'service', data.client_id);
   optionallySetFallbackData(eventData, 'entrypoint', data.entryPoint);
 
-  // The data pipeline listens on stderr.
-  process.stderr.write(JSON.stringify(eventData) + '\n');
+  if (! FLOW_METRICS_DISABLED) {
```

Comment:
You are already returning early [if FLOW_METRICS_DISABLED](https://github.com/mozilla/fxa-content-server/pull/7088/files#diff-426670954645c673de319a60a241f7c4R87) is true, no need to do it here too.